eq-author-api/middleware/loadQuestionnaire.js

```diff
@@ -1,9 +1,21 @@
-const loadQuestionnaire = require("../utils/loadQuestionnaire");
+const { QuestionnanaireVersionsModel } = require("../db/models/DynamoDB");
```

Comment:
I'd like for the models to be imported without the rest of the app knowing its Dynamo backing them and hence get it out of the path.

Reply:
Yeh I refactored most of this out but missed this one.